frontend/src/scenes/actions/EventName.js

```diff
@@ -4,7 +4,7 @@ import { useValues } from 'kea'
 import { userLogic } from 'scenes/userLogic'
 
 export function EventName({ value, onChange }) {
-    const { eventNamesGrouped } = useValues(userLogic)
+    const { eventNamesGrouped, eventNames } = useValues(userLogic)
```

Comment:
Small mismatch between these two lists throughout the component. The disabled prop only relies on eventNames. The custom event prompt shows up even when there are events you can use to filter which makes using the filter on the event table somewhat confusing. Also, now you can select a default event on the action step event filter. Not sure if this is intentional or not

Reply:
Fixed! Ready for re-qa

Reply:
PS you're currently also able to select default PostHog events. I think that's fine, there might be a usecase for having an action on pageleaves for example